psi-ms.obo

```diff
@@ -24086,6 +24098,33 @@ relationship: has_domain MS:1002305 ! value between 0 and 1 inclusive
 relationship: has_order MS:1002109 ! lower score better
 synonym: "PG: Contaminants" NARROW []
 
+[Term]
+id: MS:4000180
+name: missed cleavages histogram
```

Comment:
I would rename this to "number of missed cleavages" instead of "histogram".

Reply:
see updates